content/02.main-text.md

```diff
@@ -222,6 +218,12 @@ manubot cite --render --format=markdown \
 
 ## Authorship
 
+Manubot does not impose any restrictions on authorship.
+It allows authors to adhere to the author inclusion and author ordering conventions of their field.
+Some Manubot projects create a table in their GitHub repository to track contributors who did not commit text to the manuscript.
```

Comment:
Would linking to an example table be helpful here?

Reply:
Yes, I was thinking of that one.  I'll link it during my revisions.

Reply:
Added in 9ec30a8.  Do you think I should name the `deep-rules` project or just keep the link?